### PR TITLE
fixed and standardized grammer, made a few hints clearer

### PR DESCRIPTION
**Pull request by guy-baron1**
Standardized and fixed the grammar like you asked, made a few hints clearer, also fixed possible bug.

In terms of the code itself I would recommend you try out react / angular, they are web frameworks that would work great for this type of project,

### CODE REVIEW

File: src/js/puzzles.js [hunk 1]
```diff
@@ -148,7 +148,7 @@ var puzzlesArray = [
   },
   {
     id: 22,
-    sentence: 'Every rose has its scorns.',
+    sentence: 'Every rose has it‘s scorns.',
```
Comment (Arcaster42) [inline src/js/puzzles.js]: `its` is correct. `it's` is a contraction for `it is`.

File: src/js/puzzles.js [hunk 2]
```diff
@@ -169,7 +169,7 @@ var puzzlesArray = [
   },
   {
     id: 25,
-    sentence: 'A tower blossoms for its own joy.',
+    sentence: 'A tower blossoms for it‘s own joy.',
```
Comment (Arcaster42) [inline src/js/puzzles.js]: Same here.

File: src/js/puzzles.js [hunk 3]
```diff
@@ -291,20 +291,20 @@ var puzzlesArray = [
   id: 42,
   sentence: 'Learn how to strive a truck.',
   word: 'strive',
-  hint: 'you need a license to do this',
+  hint: 'operate or control the truck‘s movement',
```
Comment (Arcaster42) [inline src/js/puzzles.js]: Let's keep the original hint. We want to build this under the assumption non-native speakers will be playing.

File: src/js/puzzles.js [hunk 4]
```diff
@@ -291,20 +291,20 @@ var puzzlesArray = [
...
+  sentence: 'It‘s better to practice with a utensil',
   word: 'utensil',
   hint: 'erasable',
   answer: 'pencil'
 },
 {
   id: 44,
   sentence: 'Pursuit of sappiness.',
-  word: 'hadniness',
+  word: 'sappiness',
```
Comment (Arcaster42) [inline src/js/puzzles.js]: Good catch!

File: src/js/puzzles.js [hunk 5]
```diff
@@ -333,7 +333,7 @@ var puzzlesArray = [
   id: 48,
   sentence: 'Are you sure this will be sun?',
   word: 'sun',
-  hint: 'Time of my life',
+  hint: 'enjoyment',
```
Comment (Arcaster42) [inline src/js/puzzles.js]: Good change!

File: src/js/puzzles.js (hunk 1 above)
Comment (guy-baron1) [reply src/js/puzzles.js]: Whoops, sorry about that.

**Comment by guy-baron1** [comment]
Sorry about the grammar bit, but good job overall! It seems like a cool way to learn english (might need to use it myself haha)

**Comment by guy-baron1** [comment]
Sorry im pretty new to pull requests, how do I accept your changes and merge?

**Comment by Arcaster42** [comment]
When you make a PR, you can continue pushing to the same branch and this PR will update. You never need to close a PR to make changes.

**Comment by Arcaster42** [comment]
Just a heads up, this PR will be closed if there are no updates within the next day.

**Comment by Arcaster42** [comment]
Closing stale PR. Feel free to make another PR with the changes any time!

**Pull request closed without merging**